src/Compiler.php

```diff
@@ -147,11 +141,11 @@ public function repr($value)
         } elseif (\is_array($value)) {
             $this->raw('array(');
             $first = true;
-            foreach ($value as $key => $v) {
+            \foreach ($value as $key => $v) {
```

Comment:
Is phpcsfixer doing this ? oO 

I've never seen `\foreach` before

Reply:
Nope, is corrected!